### PR TITLE
Tac print attraction details

### PR DESCRIPTION
**Pull request by TimAConner**
Added printAttractionDetails and removeAttractionDetails.  This should be apart of the master because it is needed for the mvp.

### CODE REVIEW

File: js/view.js [hunk 1]
```diff
@@ -26,12 +26,16 @@ module.exports.highlightArea = function(ids) {
...
+    
+    $("#sidebar").on("click", ".attractionData", function(){
+        $(this).append(detail(attraction));
+    });
 };
 
-module.exports.removeAttractionDetails = function() {
-
+const removeAttractionDetails = function() {
+    $(".attractionDetails").remove();
```
Comment (DavidLarsKetch) [inline js/view.js]: Is this supposed to be targeting `.attractionData` in the `.hbs` template?

**Review by DavidLarsKetch — APPROVED** [review]
:+1:

**Review by iiimosley — COMMENTED** [review]
🏁

File: js/view.js (hunk 1 above)
Comment (TimAConner) [reply js/view.js]: Yes!  Thank you!

**Pull request merged**